### DISCUSSION TITLE
How to redraw drawn lines?

### DISCUSSION THREAD

**Discussion by genceraltan** [highfestiva/finplot#245] [Q&A]
hello, when I open the program again, I want the horizontal lines that I drew on the btcusdt symbol to be redrawn.
Where are the lines kept and how can I reach them?

**Comment by highfestiva** [comment]
The lines are kept in a list, `ax.vb.rois`. RoI = Region of Interest. Note that the list can be comprised of `FinPolyLine`, `FinEllipse` or `FinRect`. GL!
